pkg/api/types_swagger_doc.go

```diff
@@ -0,0 +1,956 @@
+		"code":    "Suggested HTTP return code for this status, 0 if not set.",
+	}
+}
+
+func (StatusCause) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":        "StatusCause provides more information about an api.Status failure, including cases when multiple errors are encountered.",
+		"reason":  "A machine-readable description of the cause of the error. If this value is empty there is no information available.",
+		"message": "A human-readable description of the cause of the error.  This field may be presented as-is to a reader.",
+		"field":   "The field of the resource that has caused this error, as named by its JSON serialization. May include dot and postfix notation for nested attributes. Arrays are zero-indexed.  Fields may appear more than once in an array of causes due to fields having multiple errors. Optional.\n\nExamples:\n  \"name\" - the field \"name\" on the current resource\n  \"items[0].name\" - the field \"name\" on the first array entry in \"items\"",
+	}
+}
+
+func (StatusDetails) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":                  "StatusDetails is a set of additional properties that MAY be set by the server to provide additional information about a response. The Reason field of a Status object defines what attributes will be set. Clients must ignore fields that do not match the defined type of each attribute, and should assume that any attribute may be empty, invalid, or under defined.",
+		"name":              "The name attribute of the resource associated with the status StatusReason (when there is a single name which can be described).",
+		"kind":              "The kind attribute of the resource associated with the status StatusReason. On some operations may differ from the requested resource Kind.",
+		"causes":            "The Causes array includes more details associated with the StatusReason failure. Not all StatusReasons may provide detailed causes.",
+		"retryAfterSeconds": "If specified, the time in seconds before the operation should be retried.",
+	}
+}
+
+func (TCPSocketAction) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":     "TCPSocketAction describes an action based on opening a socket",
+		"port": "Required: Port to connect to.",
+	}
+}
+
+func (TypeMeta) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":           "TypeMeta describes an individual object in an API response or request with strings representing the type of the object and its API schema version. Structures that are versioned or persisted should inline TypeMeta.",
+		"kind":       "Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to.",
```

Comment:
Most sentences start without repeating the name of the field. For consistency, remove the starting "Kind is". Thanks.